src/dlsproto/client/request/internal/GetRange.d

```diff
@@ -353,11 +359,47 @@ private scope class GetRangeHandler
     enum FiberSignal: ubyte
     {
         /// Resumes the `RecordStream` fiber.
-        ResumeRecordStream = ControllerSignal.max + 1,
+        ResumeRecordStream = ResumeRecordStreamValue,
         /// Tells the `Controller` to terminate.
         StopController
     }
 
+    /// Timer to be timeout waiting for the Stopped message
+    private SharedResources.RequestResources.ITimer timer;
+
+    /// Seconds to timeout the Stopped signal after
+    private const int seconds_stop_timeout = 60;
+
+    /// Milliseconds to timeout the Stopped signal after
+    private const int milliseconds_stop_timeout = 0;
+
+    /**************************************************************************
+
+        Sets the timer to stop this RoC, in case no message was received
+        from the node for a long period of time.
+
+    **************************************************************************/
+
+    private void setStoppedMessageTimeout ()
+    {
+        this.timer = this.resources.getTimer(this.seconds_stop_timeout,
+                this.milliseconds_stop_timeout,
+                &forceStopRequest);
+        this.timer.start();
```

Comment:
Should the timer be stopped/cancelled somewhere, if the request ends successfully before the timer fires?

Reply:
Yes, it is done in `RecordStream.stop`